tests/e2e-pw/utils/shopper.ts

```diff
@@ -340,7 +340,7 @@ export const emptyCart = async ( page: Page ) => {
 		coupons = await page.locator( '.woocommerce-remove-coupon' ).all();
 	}
 
-	await expect( page.locator( '.cart-empty.woocommerce-info' ) ).toHaveText(
-		'Your cart is currently empty.'
-	);
+	await expect(
```

Comment:
This didn't work for other themes' because their selectors are different. So I changed it to match with it's content.